get-mods.sh

```diff
@@ -25,7 +25,7 @@ get_special () {
 				if [ "$BRANCH" = master ] && [[ "$2" == icinga-php/* ]]; then
 					REF=snapshot/nightly
 				elif [ -n "$BRANCH" ] && git -C dockerweb2-temp show -s --oneline "$BRANCH"; then
-					REF="$BRANCH"
+					REF=HEAD
```

Comment:
Doesn't the following describe the change? If we have a branch name and that branch exists, use HEAD instead of that branch name.

That doesn't sound right to me.